.cirrus.yml

```diff
@@ -149,7 +145,6 @@ promote_task:
     #see https://xtranet-sonarsource.atlassian.net/wiki/spaces/RE/pages/2931064959/Cirrus+CI+-+How+to+migrate+away+of+GCP+promote+function
     ARTIFACTORY_PROMOTE_ACCESS_TOKEN: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-promoter access_token]
     GITHUB_TOKEN: VAULT[development/github/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-promotion token]
-    #artifacts that will have downloadable links in burgr
     ARTIFACTS: org.sonarsource.html:sonar-html-plugin:jar
```

Comment:
I believe the env variable can be removed too https://sonarsource.slack.com/archives/C04CVEU7734/p1720079386896279